src/lib/commandBarConfigs/validators.ts

```diff
@@ -94,11 +94,12 @@ export const revolveAxisValidator = async ({
   data: { [key: string]: Selections }
   context: CommandBarContext
 }): Promise<boolean | string> => {
-  if (!isSelections(context.argumentsToSubmit.selection)) {
+  if (!isSelections(context.argumentsToSubmit.sketches)) {
     return 'Unable to revolve, selections are missing'
   }
+  // Gotcha: this validation only works for the first sketch
```

Comment:
Didn't want to rotate on this any more than what was necessary to not break the existing, since dry run validations aren't really something we're sure have a future atm